hedera-mirror-rest/transactions.js

```diff
@@ -152,7 +153,7 @@ const getTransactionsOuterQuery = function (innerQuery, order) {
        LEFT OUTER JOIN t_transaction_types ttt ON ttt.proto_id = t.type
        JOIN crypto_transfer ctl ON tlist.consensus_timestamp = ctl.consensus_timestamp
        LEFT OUTER JOIN token_transfer ttl
-         ON t.type = 30
+         ON t.type = 14
```

Comment:
nit. We should have done this previously but could you make this a constant 
`const cryptoTransferTransactionType =14` and use here and below
```suggestion
         ON t.type = ${cryptoTransferTransactionType}
```

Reply:
Took it one step further and used the `const transactionTypes` in `constants.js` since it was already being pulled in.